packages/artifacts/package.json

```
@@ -1,65 +1,64 @@
 {
-  "name": "@zk-kit/artifacts",
-  "version": "1.3.1",
-  "description": "Utilities for downloading snark artifacts",
-  "scripts": {
-    "build": "pnpm run clean && rollup -c rollup.config.ts --configPlugin typescript",
-    "clean": "rimraf dist",
-    "test": "jest -c test/jest.config.ts",
-    "prepublishOnly": "pnpm run build"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/privacy-scaling-explorations/snark-artifacts.git"
-  },
-  "keywords": [
-    "snark",
-    "artifacts",
-    "circom",
-    "zk"
-  ],
-  "contributors": [
-    "cedoor <me@cedoor.dev>",
-    "sripwoud <me@sripwoud.xyz"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/privacy-scaling-explorations/snark-artifacts/issues"
-  },
-  "homepage": "https://github.com/privacy-scaling-explorations/snark-artifacts/tree/main/packages/artifacts",
-  "publishConfig": {
-    "access": "public"
-  },
-  "main": "./dist/index.node.js",
-  "module": "./dist/index.browser.js",
-  "types": "./dist/types/index.d.ts",
-  "exports": {
-    ".": {
-      "browser": "./dist/index.browser.js",
-      "default": "./dist/index.browser.js",
-      "node": {
-        "default": "./dist/index.node.js",
-        "import": "./dist/index.node.js",
-        "require": "./dist/index.node.cjs"
-      },
-      "types": "./dist/types/index.d.ts"
+    "name": "@zk-kit/artifacts",
+    "version": "1.3.2",
```

can you leave this to 1.3.1 instead.
```suggestion
    "version": "1.3.1",
```
and do a `pnpm changeset && git push` please :smile:

---

Yess done!